packages/expect/src/types.ts

```diff
@@ -21,7 +21,7 @@ export type AsyncExpectationResult = Promise<SyncExpectationResult>;
 export type ExpectationResult = SyncExpectationResult | AsyncExpectationResult;
 
 export type RawMatcherFn<T extends MatcherState = MatcherState> = {
-  (this: T, actual: any, expected: any, options?: any): ExpectationResult;
+  (this: T, actual: any, ...expected: Array<any>): ExpectationResult;
```

Comment:
Zero or more instead of one or two `expected` args of `any` type. Ah.. I can write a type test for this one. Just a moment (;

Reply:
Nope. Cannot make TS to error. Might be this change makes no difference right now.